src/Nethermind/Nethermind.Serialization.Rlp/TxDecoder.cs

```diff
@@ -12,6 +12,8 @@ namespace Nethermind.Serialization.Rlp
 {
     public class TxDecoder : TxDecoder<Transaction>, ITransactionSizeCalculator
     {
+        public const int MaxDelayedHashTxnSize = 32768;
```

Comment:
Any particular logic about picking this value? Apart it being power of 2, so works well with ArrayPool buckets.

Reply:
Contract size is 24kB and rounded up; shouldn't be anything larger but good to have cap

Reply:
Happy to set it lower, but KeccakF is expensive